fury/layout.py

```diff
@@ -120,3 +120,54 @@ def compute_positions(self, actors):
                                             self.aspect_ratio,
                                             self.dim)
         return positions
+
+
+class VerticalLayout(GridLayout):
+    """Provide functionalities for laying out actors in a vertical stack."""
+    def __init__(self, cell_padding=0, cell_shape="rect"):
+        """
+
+        Parameters
+        ----------
+        cell_padding : 2-tuple of float or float (optional)
+            Each cell will be padded according to (pad_x, pad_y) i.e.
+            horizontally and vertically. Padding is evenly distributed on each
+            side of the cell. If a single float is provided then both pad_x and
+            pad_y will have the same value.
+        cell_shape : {'rect', 'square', 'diagonal'} (optional)
+            Specifies the desired shape of every cell.
+            'rect' ensures the cells are the tightest.
+            'square' ensures the cells are as wide as high.
+            'diagonal' ensures the content of the cells can be rotated without
+            colliding with content of the neighboring cells.
+        """
+        super(VerticalLayout, self).__init__(cell_padding=cell_padding,
+                                             cell_shape=cell_shape)
+
+    def compute_positions(self, actors):
+        """Compute the 3D coordinates of some actors.
+
+        Parameters
+        ----------
+        actors : list of `vtkProp3D` objects
+            Actors to be layout in a vertical stack.
+
+        Returns
+        -------
+        list of 3-tuple
+            The computed 3D coordinates of every actors.
+        """
+        positions = []
+        shapes = self.get_cells_shape(actors)
+
+        # Add padding, if any, around every cell.
+        shapes = [np.array(self.cell_padding)/2. + s for s in shapes]
+
+        for shape in shapes:
+            if not len(positions):
+                actor_position = np.asarray([0, 0, 0])
+            else:
+                actor_position = positions[-1] + np.asarray([0, shape[1], 0])
+            positions.append(actor_position)
```

Comment:
I do not think the `if` is necessary. 

Just initialize position with `(0,0,0)` ->  `positions = [np.asarray([0, 0, 0]), ]` to avoid this comparaison in each iteration.

or I miss something...

Reply:
Hey @skoudoro, seems like a good idea. Done.